app/controllers/spots_controller.rb

```diff
@@ -1,10 +1,11 @@
 class SpotsController < ApplicationController
 
-  def index
-    #@spots = Spot.all
-  end
-  def show
-    #@spot = Spot.find(params[:id] )
-  end
+  # def index; end
+  #   #@spots = Spot.all
+  # end
+  # def show
+  #   # @spot = Spot.find(params[:id] )
+  #   # @reviews = Review.where(spot_id: params[:id])
+  # end
```

Comment:
Remove lines 4 - 9. Uncomment line 3. I understand your code works without it, but having it uncommented is more clear to the person reviewing your PR.

Reply:
Done